addons/website_profile/static/src/js/website_profile.js

```diff
@@ -42,6 +43,37 @@ sAnimations.registry.websiteProfile = sAnimations.Class.extend({
     },
 });
 
+sAnimations.registry.websiteProfileEditor = sAnimations.Class.extend({
+    selector: '.o_wprofile_editor_form',
+    read_events: {
+        'click .o_wprofile_submit_btn': '_onSubmitClick',
+    },
+
+    /**
+     * @override
+     */
+    start: function () {
+        var self = this;
+        var $textarea = this.$('textarea.load_editor');
+        var wysiwyg = new Wysiwyg(self, {
+            recordInfo: {
+                context: self._getContext(),
+                res_model: 'res.users',
+                res_id: parseInt(self.$('input[name=user_id]').val()),
+            },
+        });
+
+        if (!$textarea.val().match(/\S/)) {
+            $textarea.val('<p><br/></p>');
+        }
+        wysiwyg.attachTo($textarea);
+        return this._super.apply(this, arguments);
```

Comment:
Waiting for `attachTo` would still be better:
`return $.when(def, wysiwyg.attachTo($textarea));`